go.mod

```diff
@@ -110,7 +110,7 @@ require (
 	github.com/prometheus/client_model v0.4.0
 	github.com/prometheus/common v0.44.0
 	github.com/rancher/aks-operator v1.2.1-rc2
-	github.com/rancher/apiserver v0.0.0-20230831052300-120e615b17ba
+	github.com/rancher/apiserver v0.0.0-20240205154815-a3b9e3721c1b
```

Comment:
This is pointing to https://github.com/rancher/apiserver/commits/release/v2.8.s3 instead of https://github.com/rancher/apiserver/commits/release/v2.8, no?

Reply:
Yes, this is because `release/v2.8` has more changes (1.28 k8s support) which makes it difficult to merge. Once 1.28 k8s support is added to 2.8 (and 2.9) will be able to point back to head of those norman branches.

Reply:
Same reasoning for not bumping steve as part of this PR, it will be done later once we get 1.28/wrangler v2 in 2.8 and 2.9 head.

Reply:
Ahh, got it! Makes total sense! I'll approve them and this can be fixed later.